src/overlays/actors/ovl_Eff_Dust/z_eff_dust.c

```diff
@@ -22,28 +29,392 @@ const ActorInit Eff_Dust_InitVars = {
     (ActorFunc)EffDust_Update,
     (ActorFunc)EffDust_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Eff_Dust/func_8099D8D0.s")
+static Gfx* D_8099EB60[] = { 0xDF000000, 0x00000000, 0x00000000, 0x00000000 };
```

Comment:
Since this is a DList it can be run through gfxdis
```suggestion
static Gfx* D_8099EB60[] = { gsSPEndDisplayList() },
```